image.go

```diff
@@ -25,19 +25,24 @@ func resizeAndRotate(img image.Image, width, height int, devname string) image.I
 
 func deviceSpecifics(devName string, width, height int) (*gift.GIFT, error) {
 	switch devName {
-		case "Streamdeck XL":
-			return gift.New(
-				gift.Resize(width, height, gift.LanczosResampling),
-				gift.Rotate180(),
-			), nil
-		case "Streamdeck Mini":
-			return gift.New(
-				gift.Resize(width, height, gift.LanczosResampling),
-				gift.Rotate90(),
-				gift.FlipVertical(),
-			), nil
-		default:
-			return nil, errors.New(fmt.Sprintf("Unsupported Device: %s", devName))
+	case "Streamdeck XL":
+		return gift.New(
+			gift.Resize(width, height, gift.LanczosResampling),
+			gift.Rotate180(),
+		), nil
+	case "Streamdeck Mini":
+		return gift.New(
+			gift.Resize(width, height, gift.LanczosResampling),
+			gift.Rotate90(),
+			gift.FlipVertical(),
+		), nil
+	case "Streamdeck (original v2)":
```

Comment:
You could simplify this by updating the `Streamdeck XL` case to look like this:

```go
case "Streamdeck XL", "Streamdeck (original v2)":
		return gift.New(
			gift.Resize(width, height, gift.LanczosResampling),
			gift.Rotate180(),
		), nil
```

Reply:
you're correct, I picked up the wrong commit. I'll recreate a proper PR also following the contribution guidelines